airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py

```diff
@@ -43,42 +43,20 @@ def parse_dates(stream_slice):
     return start_date, end_date
 
 
-def get_date_params(start_date: str, time_zone=None, range_days: int = None, end_date: pendulum.datetime = None) -> Tuple[str, str]:
-    """
-    Returns `start_date` and `end_date` for the given stream_slice.
-    If (end_date - start_date) is a big date range (>= 1 month), it can take more than 2 hours to process all the records from the given slice.
-    After 2 hours next page tokens will be expired, finally resulting in page token expired error
-    Currently this method returns `start_date` and `end_date` with 15 days difference.
```

Comment:
even though we're deleting this method, we should retain the context of this comment in `chunk_date_range()` Can you update the docstring accordingly. The one chunk_date_range might be outdated and it references the Slack API.

Reply:
:+1: done